lib/core/src/lib/datatable/components/datatable/datatable.component.html

```diff
@@ -236,28 +251,29 @@
                         </button>
                     </mat-menu>
                 </div>
-
-                <label *ngIf="multiselect"
-                        (keydown.enter)="onEnterKeyPressed(row, $any($event))"
-                        (click)="onCheckboxLabelClick(row, $event)"
-                        [for]="'select-file-' + idx"
-                        class="adf-datatable-cell adf-datatable-checkbox adf-datatable-checkbox-single"
-                        tabindex="0">
+                <td
```

Comment:
This is only the checkbox part not the whole cell so I don't think this is a valid change